### PR TITLE
Add test suite filtering, minor fixes

### PR DESCRIPTION
**Pull request by naushniki**
-add entrypoint script run_test_suites.py
-fix cleanup.py to work with src installer
-fix infinite loop in starter manager

### CODE REVIEW

**Comment by naushniki** [comment]
@MDAlFattah Please check that UI tests are not broken on this branch.

**Comment by naushniki** [comment]
https://jenkins.arangodb.biz/view/Upgrade/job/Upgrade-3.8-to-3.9.Windows/448/
https://jenkins.arangodb.biz/view/Upgrade/job/Upgrade-3.9-to-devel.Linux/384/

File: release_tester/test_driver.py [hunk 1]
```diff
@@ -589,3 +598,49 @@ def run_license_manager_tests(
...
+        results.extend(self.run_conflict_tests(versions, True))
+        results.extend(self.run_conflict_tests(versions, False))
+        results.extend(self.run_license_manager_tests(versions))
+        return results
+
+    def run_test_suites(self, versions: list, include_suites=(), exclude_suites=()):
+        suite_classes=[]
+        if len(include_suites)==0 and len(exclude_suites)==0:
+            return self.run_all_test_suites(versions)
+        elif len(include_suites)>0 and len(exclude_suites)==0:
```
Comment (dothebart) [inline release_tester/test_driver.py:614]: everywhere else in this PR you have been reformatting ==/> to contain blanks around them? ;-)

**Review by dothebart — APPROVED** [review]
if you could find a good solution to reduce the WET, that'd be great.
Rest LGTM.

File: release_tester/test_driver.py (hunk 1 above)
Comment (naushniki) [reply release_tester/test_driver.py]: I had to correct this part manually. I don't know why black did not take care of this.

**Pull request merged**